icepyx/core/query.py

```diff
@@ -136,12 +136,7 @@ def __init__(
             self._temporal = tp.Temporal(date_range, start_time, end_time)
 
     def __str__(self):
-        str = "Extent type: {0} \nCoordinates: {1}\nDate range: ({2}, {3})".format(
-            self._spatial._ext_type,
-            self._spatial._spatial_ext,
-            self._temporal._start,
-            self._temporal._end,
-        )
+        str = f"Extent type: {self._spatial._ext_type} \nCoordinates: {self._spatial._spatial_ext}\nDate range: ({self._temporal._start}, {self._temporal._end})"
```

Comment:
I personally find this a bit less readable. What about:
```suggestion
        str = (
            f"Extent type: {self._spatial._ext_type}"
            f"\nCoordinates: {self._spatial._spatial_ext}"
            f"\nDate range: ({self._temporal._start}, {self._temporal._end})"
        )
```

Reply:
Funny that a repo that fails to set a ruff `line-length` wants to wrap lines.

Reply:
`E501` is the ruff rule that enables `line-length` for the linter.  It is universally ignored in this repo.

Reply:
That's not quite a complete description of the situation. `E501` is the "line too long rule", true. However, Ruff is a linter and formatter, and uses `line-length` for both, but `E501` only affects linting behavior. The reason why E501 was disabled is explained in the config file in a co-located comment. Please try for yourself! If you add a long line, the formatter will work as described above! This is the formatting that was applied for me:

```
-print("This is a very very very very very very very very very very very long line" + "And I want it to be formatted")
+print(
+    "This is a very very very very very very very very very very very long line"
+    + "And I want it to be formatted"
+)
```